pi5/board.go

```diff
@@ -36,6 +35,7 @@ func init() {
 }
 
 // pins are stored in /dev/gpiomem in order of gpio nums, so we must convert from pin name (physical num) to GPIO number.
+// these are redundant with gpioMappings from gl.GetGPIOBoardMappings/data.go
 var pinNameToGPIONum = map[string]int{
 	"3":  2,
```

Comment:
just noticed these are the same information, but we use data.go to stand up our gpio pins/interrupts while we use this map for standing up the pulls. Would it be okay to remove this list in a future pr in favor of only using `data.go`, since we have more information there? using data.go would also "fix" our issue with hardwarer pwm